From a8ecd8ae386b609d7da2d500ef8e9baec8dd7dc7 Mon Sep 17 00:00:00 2001
From: Jan Stancek <jstancek@redhat.com>
Date: Fri, 9 Mar 2018 10:58:01 +0100
Subject: [PATCH 1/2] mallocstress: extend test time

Arm systems with lot of memory and THP enabled are
hitting a timeout at ~5 minute mark as reported here:
  mallocstress poor performance with THP on arm64 system
  https://marc.info/?l=linux-mm&m=151864950330870&w=2

Some other systems are already getting close to ~5 minutes,
so let's extend it to 10m for all.

Signed-off-by: Jan Stancek <jstancek@redhat.com>
Acked-by: Cyril Hrubis <chrubis@suse.cz>
---
 testcases/kernel/mem/mtest07/mallocstress.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/testcases/kernel/mem/mtest07/mallocstress.c b/testcases/kernel/mem/mtest07/mallocstress.c
index 2e900ad..70b614b 100644
--- a/testcases/kernel/mem/mtest07/mallocstress.c
+++ b/testcases/kernel/mem/mtest07/mallocstress.c
@@ -243,6 +243,7 @@ static void cleanup(void)
 }
 
 static struct tst_test test = {
+	.timeout = 600,
 	.needs_checkpoints = 1,
 	.setup = setup,
 	.cleanup = cleanup,
-- 
2.9.3

